make.go

```diff
@@ -821,10 +821,10 @@ func execMake(args []string) {
 	log.Printf("    git add debian && git commit -a -m 'Initial packaging'\n")
 	log.Printf("    gbp buildpackage --git-pbuilder\n")
 	log.Printf("\n")
-	log.Printf("To create the packaging git repository on alioth, use:\n")
-	log.Printf("    ssh git.debian.org \"/git/pkg-go/setup-repository %s 'Packaging for %s'\"\n", debsrc, debsrc)
+	log.Printf("To create the packaging git repository on salsa, use:\n")
+	log.Printf("    SALSA_TOKEN=\"<token here>\" dh-make-golang create-salsa-project %s", debsrc)
```

Comment:
The SALSA_TOKEN can be removed now that create-salsa-project is done server-side.

Reply:
done!